.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     <<: *defaults
     steps:
       - checkout
-      - run: ./gradlew clean build --stacktrace
+      - run: ./gradlew clean build --stacktrace -Dhttp.socketTimeout=20000 -Dhttp.connectionTimeout=20000
```

Comment:
You can specify `-D` command-line options in [`gradle.properties`](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_system_properties). I imagine we'll have other overrides for our builds in the future. Thoughts?

Reply:
Ah nice, yeah that's better